components/Search/forms/SearchCasesForm.jsx

```diff
@@ -120,6 +122,26 @@ const SearchCasesForm = ({ onFormSubmit, defaultValues }) => {
       </div>
 
       <Button label="Search" type="submit" disabled={!isDirty} />
+      <input
+        className="govuk-body"
+        type="reset"
```

Comment:
Let's make it a `span` (with `role="button"`) so we are sure it's not conflicting with the `form` itself.
Also, it will prevent the necessity of override a bunch of styles 👎 )